### DISCUSSION TITLE
13. Spring 에서 Interceptor와 Servlet Filter에 대해 설명해 주세요.

### DISCUSSION THREAD

**Discussion by KIMSEI1124** [Study-Diary/Computer-Science#50] [Spring]
설명만 들어보면 인터셉터만 쓰는게 나아보이는데, 아닌가요? 필터는 어떤 상황에 사용 해야 하나요?

**Comment by jiwon83** [comment]
doFilter의 경우, Controller 호출 전후에  두 번 실행하나요?

**Comment by jiwon83** [comment]
jwt 처리를 필터랑 인터셉트를 어디에 사용하면 좋을까?
